docs/development/input-processing.md

```diff
@@ -569,34 +571,9 @@ be a tool to quickly and easily derive meaningful data from primitive input but
 will also require additions to the [Data-library](../../src/Data) and the components
 that use their own data structures.
 
-
-### Validation
-
-The [Validation-library](../../src/Validation) provides an abstraction for
-`Constraints` on data, where a `Constraint` is a check in conjunction with a
-builder for a human readable error-message. Like the [Transformation-library](../../src/Refinery/Transformation)
-the [Validation-library](../../src/Validation) also attempts to provide a set of
-standard constraints and facilities to compose complex constraints from simpler
-ones.
-
-Currently the library provides 14 prefactored constraints and one constraint
-that allows to create custom constraints via closures. Here we certainly
-require additions to the currently available constraints as well as a better
-internal structure that e.g. groups the constraints by the type they are acting
-upon.
-
-Other then the two aforementioned libraries, the [Validation-library](../../src/Validation)
-mostly will deal with constraints derived from policies on the input data. It
-thus will certainly need backup from those two libraries regarding the structures
-it operates upon. Like the `Transformations` the `Constraints` offer the possibility
-to declaratively define these constraints and thus make them understandable and
-readable. To fully unfold this potential the general use of custom constraints
-needs to be minimal.
-
-This also offers a perspective how other policy enforcing components in the system
-may come into play here: Like the [Validation-library](../../src/Validation) itself
-they may offer sets of constraints regarding the policies they are enforcing.
-
+The [Refinery-library](../../src/Refinery) also offers a perspective how other policy
```

Comment:
I think this should be "...a perspective on how other components that enforce policies..."

Reply:
+1